filter/expand_list.go

```diff
@@ -119,6 +119,11 @@ func (f *ExpandList) Process(l baker.Record, next func(baker.Record)) {
 	atomic.AddInt64(&f.numProcessedLines, 1)
 
 	list := l.Get(f.source)
+	if len(list) == 0 {
+		next(l)
```

Comment:
One can understand why we early return here only if they know the behavior of `bytes.Split`, I'd add a small comment to help future readers here